### ISSUE TITLE
[Create Event] user cannot select 'Event type' using the 'Tab' key,

### ISSUE THREAD

**Issue by Tetiana1202** [ita-social-projects/GreenCity#6843]
Environment: MacOS 13.5.1, Google Chrome (120.0.6099.129) 
Reproducible: always
Build found: 07.01.2024

Preconditions:
 1. User is Authorized
 2. The "Events" page is open.

Steps to reproduce
1. Click on 'Create Event' button
2. Use the "Tab" key to go to "Event Type", namely to the "Close" button.
3. Press the 'Enter' key

Actual result:
1. The "Close" button is not active. The 'Open' button is active

Expected result:
1. The "Close" button is active, highlighted in green. The 'Open' button is not active
